src/properties.rs

```diff
@@ -71,12 +71,16 @@ impl Property {
     }
 
     /// Writes this Property to `out`
-    pub fn fmt_write<W: fmt::Write>(&self, out: &mut W) -> Result<(), fmt::Error> {
-        try!(write!(out, "{}", self.key));
+    pub fn fmt_write<W: Write>(&self, out: &mut W) -> Result<(), fmt::Error> {
+        // A nice starting capacity for the majority of content lines
+        let mut line = String::with_capacity(150);
```

Comment:
In Calendar.print()
```rust    
let mut out = String::new();
```
It doesn't set any default size so it probably would have reallocations.
For any other ```out``` (usually a String), it depends if the caller to ```fmt_write()``` allocated ```out``` or not. For the line allocation, the final size of ```out``` is not determined and doesn't have a method to access its length.

Most content lines seem quite short, so I thought this would be a nice size for most cases.